app/assets/stylesheets/media.scss

```diff
@@ -125,7 +125,12 @@
         border: none;
       }
 
+      #sul-embed-purl-link {
+        width: calc(100% - 2rem);
```

Comment:
Can this width go in  https://github.com/sul-dlss/sul-embed/blob/f2e2d8fb207c4cc1085b22a472d9b687f764a8ac/app/assets/stylesheets/media.scss#L118

Reply:
@jcoyne yeah, that works, done.